docs/resources/user.mdx

```diff
@@ -110,6 +117,15 @@ Premium types denote the level of premium a user has. Visit the [Nitro](https://
 | 2     | Nitro         |
 | 3     | Nitro Basic   |
 
+###### User Primary Guild
+
+| Field             | Type       | Description                                                   |
+|-------------------|------------|---------------------------------------------------------------|
+| identity_guild_id | ?snowflake | the id of the user's primary guild                            |
+| identity_enabled  | ?boolean   | whether the user is displaying the primary guild's server tag |
```

Comment:
I really do think the behaviour of this being None meaning a new identity hasn't been affirmed should be documented. It is very unclear otherwise.